pyserver/main.py

```diff
@@ -81,12 +82,15 @@ def comments_to_tree(req: CommentsLLMConfig, log_to_wandb:bool = False):
     },
     "comments": [
         {
+            "id: "..",
```

Comment:
nit: prefer literal example in comments, so "id" : "123", then "id" : "456", "id" : "789". Can even put in an example like "id" : "my-random-string-here" to make it super clear to the reader. Also "id" is missing the closing quote here in all the rows.